doors_dashboards/components/scatterplot.py

```diff
@@ -518,143 +558,160 @@ def register_callbacks(self, component_ids: List[str], dashboard_id: str = None)
+                selected_group_dropdown_id
+            ].children
+            temp = {GROUP: selected_group_item}
             return temp
 
         @callback(
             Output(TEMP_STORE_ID, "data", allow_duplicate=True),
-            [Input(point_x_drop_id, 'n_clicks_timestamp')
-             for point_x_drop_id in point_x_drop_options],
-            prevent_initial_call=True
+            [
+                Input(point_x_drop_id, "n_clicks_timestamp")
+                for point_x_drop_id in point_x_drop_options
+            ],
+            prevent_initial_call=True,
         )
         def selector_to_temp_xvar(*timestamps):
             if not any(timestamps):
                 return dash.no_update
             latest_timestamp_index = timestamps.index(
-                max(t for t in timestamps if t is not None))
-            xvar_drop_option_id = \
-                list(self.xvar_drop_options.keys())[latest_timestamp_index]
-            xvar_drop_option_value = \
-                self.xvar_drop_options[xvar_drop_option_id].children
-            temp_data = {
-                X_VARIABLE: xvar_drop_option_value
-            }
+                max(t for t in timestamps if t is not None)
+            )
+            xvar_drop_option_id = list(self.xvar_drop_options.keys())[
+                latest_timestamp_index
+            ]
+            xvar_drop_option_value = self.xvar_drop_options[
+                xvar_drop_option_id
+            ].children
+            temp_data = {X_VARIABLE: xvar_drop_option_value}
             return temp_data
 
         @callback(
             Output(TEMP_STORE_ID, "data", allow_duplicate=True),
-            [Input(point_y_drop_id, 'n_clicks_timestamp')
-             for point_y_drop_id in point_y_drop_options],
-            prevent_initial_call=True
+            [
+                Input(point_y_drop_id, "n_clicks_timestamp")
+                for point_y_drop_id in point_y_drop_options
+            ],
+            prevent_initial_call=True,
         )
         def selector_to_temp_yvar(*timestamps):
             if not any(timestamps):
                 return dash.no_update
             latest_timestamp_index = timestamps.index(
-                max(t for t in timestamps if t is not None))
-            yvar_drop_option_id = \
-                list(self.point_y_drop_options.keys())[latest_timestamp_index]
-            yvar_drop_option_value = \
-                self.point_y_drop_options[yvar_drop_option_id].children
-            temp_data = {
-                Y_VARIABLE: yvar_drop_option_value
-            }
+                max(t for t in timestamps if t is not None)
+            )
+            yvar_drop_option_id = list(self.point_y_drop_options.keys())[
+                latest_timestamp_index
+            ]
+            yvar_drop_option_value = self.point_y_drop_options[
+                yvar_drop_option_id
+            ].children
+            temp_data = {Y_VARIABLE: yvar_drop_option_value}
```

Comment:
![image](https://github.com/bcdev/doors-dashboards/assets/144326622/244ce138-54f1-441e-be76-efd9f5c7ef6c)

![image](https://github.com/bcdev/doors-dashboards/assets/144326622/b59c07d1-9d76-4c49-b30c-71d1f53c0ee1)


When I am changing collection the y variable dropdown label is not updated